api/serializers.py

```diff
@@ -73,9 +73,9 @@ class VolunteerSerializer(serializers.ModelSerializer):
     class Meta:
         model = Volunteer
         fields = ['id', 'first_name', 'last_name', 'tz_number', 'phone_number', 'date_of_birth', 'age',
-                  'gender', 'city', 'address', 'areas', 'organization', 'moving_way',
-                  'week_assignments_capacity', 'wanted_assignments', 'email', 'email_verified', 'score',
-                  'created_date', 'times_volunteered', 'languages']
+                  'gender', 'city', 'address', 'organization', 'moving_way', 'week_assignments_capacity',
+                  'wanted_assignments', 'email', 'email_verified', 'score', 'created_date', 'times_volunteered',
```

Comment:
Times volunteered queries one time per volunteer. Please change it to the bulk query (there's a function in the manager for that).